docs/user/alerting/pre-configured-connectors.asciidoc

```diff
@@ -85,3 +112,19 @@ A message indicates that this is a preconfigured connector.
 
 [role="screenshot"]
 image::images/pre-configured-connectors-view-screen.png[Pre-configured connector view details]
+
+The connector details preview is disabled for preconfigured connectors.
+
+[role="screenshot"]
+image::images/pre-configured-action-type-managing.png[Connectors managing tab with pre-cofigured]
+
+
+[float]
+[[managing-pre-configured-action-types]]
+=== Managing preconfigured action types
+
+Clicking *Create connector* shows the list of available action types.
+Disabled action types are not included.
+
+[role="screenshot"]
+image::images/pre-configured-action-type-select-type.png[Pre-configured connector create menu]
```

Comment:
Suggest cutting off the bottom of this image as its just a lot of white space.

Suggest making the sections from "Alert with a preconfigured connector" on down more of a workflow.  Happy to discuss with you.  We can do this in a separate PR.